.thoth.yaml

```diff
@@ -22,6 +22,7 @@ managers:
         - goern
         - harshad16
         - pacospace
+        - Gkrumbach07
```

Comment:
As Kebechet can take the information about maintainers from the OWNERS (approvers), what about removing the `maintainers` section from `.thoth.yaml` and rely on the approvers from OWNERS?

This would avoid redundancy and potential inconsistencies...

Reply:
Sounds good to me, do we have this information stated somewhere in core?

Reply:
The documentation is in the [Kebechet docs for the version manager](https://thoth-station.ninja/docs/developers/kebechet/managers/version.html#configuration).

Maybe it could be expanded and made more visible?

It's unclear in the docs which takes precedence, but I would expect `.thoth.yaml` to override OWNERS (and so seems to do the [code](https://github.com/thoth-station/kebechet/blob/e9e2cab2e232a7d8a170ad6ac072154c055895ce/kebechet/managers/version/version.py#L289)). The doc also wrongly mentions a `maintainers` key within the `OWNERS` file, when it's actually `approvers` there... will open an issue to improve the docs